build/docker/server/Dockerfile.el9

```diff
@@ -4,7 +4,6 @@ ARG VERSION
 ARG BUILD_DATE
 
 ENV LANG=en_US.utf8
-ENV LC_ALL=en_US.utf8
```

Comment:
Are you sure that we don't need it anymore? I remember there were issue with PostgreSQL. I hope it won't affect upgrade from PMM 2 to PMM 3.